### PR TITLE
feat(NF: 775): add values field for EditableMetadata

### PR DESCRIPTION
**Pull request by anamariamarian1**

### CODE REVIEW

File: neverpile-fusion-core/src/main/java/com/neverpile/fusion/model/EditableMetadata.java [hunk 1]
```diff
@@ -36,6 +36,15 @@ public void setEdit(EditableMetadataEntry[] edit) {
   public static class EditableMetadataEntry {
     String key;
     String label;
+    String[] values;
```
Comment (gehleda) [inline neverpile-fusion-core/src/main/java/com/neverpile/fusion/model/EditableMetadata.java:39]: "values" or "allowedValues"? What fits better here?

File: neverpile-fusion-core/src/main/java/com/neverpile/fusion/model/EditableMetadata.java [hunk 2]
```diff
@@ -36,6 +36,15 @@ public void setEdit(EditableMetadataEntry[] edit) {
   public static class EditableMetadataEntry {
```
Comment (gehleda) [inline neverpile-fusion-core/src/main/java/com/neverpile/fusion/model/EditableMetadata.java:36]: Javadoc should also be adapted to the current extension (allowing to specify allowed values).

File: neverpile-fusion-core/src/main/java/com/neverpile/fusion/model/EditableMetadata.java (hunk 1 above)
Comment (Crow08) [reply neverpile-fusion-core/src/main/java/com/neverpile/fusion/model/EditableMetadata.java:39]: maybe "options" 
(this is the terminology in html select)

**Review by Crow08 — APPROVED** [review]

**Review by ebidosm — APPROVED** [review]
don't forget to fix the failed tests

**Pull request merged**